Source/Navigation/Web.React/lib/index.js

```diff
@@ -94,6 +94,9 @@ class CBSNavigation extends Component {
       // TODO: mark the dropdown elements as well as the main one
     }
 
+    if (active == '') {// admin case
```

Comment:
Should this also set admin as active as in the other index.js-file or is this correct?